example/src/App.tsx

```diff
@@ -20,7 +20,7 @@ import {
   DEFAULT_DIDCOMM_SERVICE_UUID,
   DEFAULT_DIDCOMM_MESSAGE_CHARACTERISTIC_UUID,
   DEFAULT_DIDCOMM_INDICATE_CHARACTERISTIC_UUID,
-} from '@animo-id/react-native-ble-didcomm'
+} from '../../src'
```

Comment:
This should still be `@animo-id/react-native-ble-didcomm` it will find it via a symlink, eventhough the types dont work

Reply:
Ohh. Got it.